packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -972,6 +972,22 @@ function boundAndSnapToPolygonAngle(
 ) {
     const startingPoint = coords[index];
 
+    return calculateAngleSnap(
+        destinationPoint,
+        range,
+        coords,
+        index,
+        startingPoint,
+    ) as vec.Vector2;
```

Comment:
Generally we'd want to avoid `as` since that blows away typechecking and. If you need to keep it, you can add a comment explaining why.

```suggestion
    ) satisfies vec.Vector2;
```